requirements.txt

```diff
@@ -1,4 +1,3 @@
-configparser>=3.5.0
-fuzzfetch==0.5.7
--e git://github.com/MozillaSecurity/lithium@master#egg=lithium-reducer
```

Comment:
The #egg format is specifically required in order to use pipenv.  Unless you see a reason not to use them, I think we should revert this change.

Reply:
Ok, I'll revert this portion. However, please ensure this still works for you using pipenv.